packages/@azure/servicebus/data-plane/test/namespace.spec.ts

```diff
@@ -372,13 +375,10 @@ describe("Test createFromAadTokenCredentials", function(): void {
 
   it("throws error for an invalid host", async function(): Promise<void> {
     const env = getEnvVars();
-    tokenCreds = await loginWithServicePrincipalSecret(
-      env.clientId,
-      env.clientSecret,
-      env.tenantId,
-      {
+    tokenCreds = <any>(
```

Comment:
Is this because of https://github.com/Azure/ms-rest-nodeauth/issues/49?

Reply:
Indeed.

Reply:
We don't need the casting after updating to use 0.9.2 of ms-rest-nodeauth